File: 0169 majority-element/solution.java
class Solution {
    public int majorityElement(int[] nums) {
      int c1=0,c2=0;
      for(int i:nums) {
        if(c1==0) c2=i;
        c1+=c2==i ? 1 : -1;
      }
      return c2;
}}

File: 0169 majority-element/solution.cs
public class Solution {
    public int MajorityElement(int[] nums) {
        int res=0,count=0;
        foreach(var i in nums) {
            if(count==0) res=i;
            count+=res==i ? 1 : -1;
        }
        return res;
    }
}
